### PR TITLE
add note to check compiler version

### PR DESCRIPTION
**Pull request by rencryptofish**

### CODE REVIEW

File: docs/develop/node/archival/run-archival-node-without-nearup.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Running an archival node is very similar to running a [validator node](/docs/dev
 
 ## Prerequisites {#prerequisites}
 
-- [Rust](https://www.rust-lang.org/). If not already installed, please [follow these instructions](https://docs.near.org/docs/tutorials/contracts/intro-to-rust#3-step-rust-installation).
+- [Rust](https://www.rust-lang.org/). If not already installed, please [follow these instructions](https://docs.near.org/docs/tutorials/contracts/intro-to-rust#3-step-rust-installation). If already installed, please note the compiler version in [rust-toolchain](https://github.com/near/nearcore/blob/master/rust-toolchain.toml)
```
Comment (matklad) [inline docs/develop/node/archival/run-archival-node-without-nearup.md:18]: I'd rephrase it along the following lines

```
Latest stable version of Rust. We recommend using https://rustup.rs to manage installation
```

That is:

* latest stable, rather than `rust-toolchain.toml`. We generally support latest stable and this additionally gives the info "you probably won't be able to use outdated rust from your linux distro"
* don't link to *our* docs on how to install Rust, link to the Rust docs instead.

**Review by bucanero — CHANGES_REQUESTED** [review]
Hi @rencryptofish , thanks for the PR.
Could you check the comments left by @matklad ? thanks!

**Comment by bucanero** [comment]
Node documentation has moved to https://github.com/near/node-docs
